src/components/fieldset/Fieldset.stories.md

```diff
@@ -0,0 +1 @@
+`import Fieldset from 'box-ui-elements/es/components/fieldset/Fieldset;`
```

Comment:
Drop the last "Fieldset" so that we use default export of the folder

"import Fieldset from 'box-ui-elements/es/components/fieldset"